source/server-discovery-and-monitoring/server-discovery-and-monitoring.rst

```diff
@@ -798,6 +818,13 @@ MUST be replaced with the new ServerDescription.
 
 .. _is compatible:
 
+
+TopologyType LoadBalanced
+`````````````````````````
+
+See the `Load Balancer Specification <https://github.com/mongodb/specifications/blob/master/source/load-balancers/load-balancers.rst#server-discovery-and-monitoring>`__ for details.
```

Comment:
This should be a relative URI for consistency: `../load-balancers/load-balancers.rst#server-discovery-and-monitoring`.

Reply:
Updated to relative URI.